<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamListWidget</class>
 <widget class="QWidget" name="StreamListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QTreeView" name="resultStreamView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_StreamAdd">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icons/navigate_right.png</normaloff>:/icons/navigate_right.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_StreamAddAll">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icons/navigate_right2.png</normaloff>:/icons/navigate_right2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_StreamRemoveAll">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icons/navigate_left2.png</normaloff>:/icons/navigate_left2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_StreamRemove">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icons/navigate_left.png</normaloff>:/icons/navigate_left.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTreeView" name="seedlinkStreamView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="lineEdit_Stream">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>NET.STA.CHAN.LOC.D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Add">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icons/navigate_plus.png</normaloff>:/icons/navigate_plus.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Refresh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Fetch server list</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_RefreshCancel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>seedlinkStreamView</tabstop>
  <tabstop>resultStreamView</tabstop>
  <tabstop>toolButton_StreamAdd</tabstop>
  <tabstop>toolButton_StreamAddAll</tabstop>
  <tabstop>toolButton_StreamRemoveAll</tabstop>
  <tabstop>toolButton_StreamRemove</tabstop>
  <tabstop>lineEdit_Stream</tabstop>
  <tabstop>toolButton_Add</tabstop>
 </tabstops>
 <resources>
  <include location="../rc/rc.qrc"/>
 </resources>
 <connections/>
</ui>
